ansible/playbooks/adhoc/hotfix/roles/pre_update_node/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Choose a single master to run commands on
+  set_fact:
+    cluster_master: "{{ groups['oo_hosttype_master'] | intersect(groups['oo_clusterid_' ~ oo_clusterid]) | first }}"
+  run_once: true
+  when: "{{ update_needed == True }}" 
+
+- name: Make node unschedulable
+  command: "oadm manage-node {{ hostvars[oo_name].ec2_private_dns_name }} --schedulable=false"
```

Comment:
consider using the oadm_manage_node module that is available.

Because of time constraints, consider adding a FIXME so this can get fixed/updated in future versions.